pkg/ddevapp/typo3.go

```diff
@@ -29,28 +32,27 @@ $GLOBALS['TYPO3_CONF_VARS'] = array_replace_recursive(
                     'password' => 'db',
                     'port'     => '3306',
                     'user'     => 'db',
-                ]
-            ]
+                ],
+            ],
         ],
         // This GFX configuration allows processing by installed ImageMagick 6
         'GFX' => [
             'processor' => 'ImageMagick',
             'processor_path' => '/usr/bin/',
-            'processor_path_lzw' => '/usr/bin/'
+            'processor_path_lzw' => '/usr/bin/',
         ],
         // This mail configuration sends all emails to mailhog
         'MAIL' => [
             'transport' => 'smtp',
-            'transport_smtp_server' => 'localhost:1025'
+            'transport_smtp_server' => 'localhost:1025',
         ],
         'SYS' => [
             'trustedHostsPattern' => '.*.*',
             'devIPmask' => '*',
-            'displayErrors' => 1
-        ]
+            'displayErrors' => 1,
+        ],
```

Comment:
btw this trailing comma is recommended because it's less error prone if you add something new afterwards

Reply:
Always. But of course they ruin js/json :)